db/models/categories.model.js

```diff
@@ -0,0 +1,5 @@
+const db = require("../connection")
+
+exports.fetchCategories = () => {
+    return db.query('SELECT * FROM categories;')
```

Comment:
It's probably easiest to achieve the change by adding a then block to this query to return the array on it's own